homeassistant/components/sensor/hydroquebec.py

```diff
@@ -115,7 +116,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     for variable in config[CONF_MONITORED_VARIABLES]:
         sensors.append(HydroQuebecSensor(hydroquebec_data, variable, name))
 
-    add_devices(sensors, True)
```

Comment:
How is the `update` call useless?

Reply:
It's just to speed up the startup of hass, the update can takes 10-20 seconds, and it's not needed to create sensors component in hass. So sensors will be updated later after the startup.

Reply:
Ok.